### PR TITLE
Populated Chapter 10 lessons

### PR DESCRIPTION
**Pull request by 0tuedon**

### CODE REVIEW

File: content/lessons/chapter-10/making-a-payment-6.tsx [hunk 1]
```diff
@@ -1,51 +1,24 @@
...
-  image: '/assets/images/chapter-9-intro-2.jpg',
+  title: 'chapter_ten.making_a_payment_seven.title',
+  navigation_title: 'chapter_ten.making_a_payment_seven.nav_title',
+  image: '/assets/images/chapter-10-opening-a-channel-3.jpg',
   theme: 'bg-[#401d4e]',
   key: 'CH10MAP6',
 }
 
-export default function MakingAPayment6({ lang }) {
+export default function MakingAPayment7({ lang }) {
```
Comment (benalleng) [inline content/lessons/chapter-10/making-a-payment-6.tsx:14]:
```suggestion
export default function MakingAPayment6({ lang }) {
```

File: content/lessons/chapter-10/making-a-payment-7.tsx [hunk 2]
```diff
@@ -1,30 +1,87 @@
...
-  navigation_title: 'chapter_ten.intro_one.nav_title',
-  image: '/assets/images/chapter-10-intro-1.jpg',
+  title: 'chapter_ten.making_a_payment_eight.title',
+  navigation_title: 'chapter_ten.making_a_payment_eight.nav_title',
   theme: 'bg-[#401d4e]',
   key: 'CH10MAP7',
 }
 
-export default function MakingAPayment7({ lang }) {
+export default function MakingAPayment8({ lang }) {
```
Comment (benalleng) [inline content/lessons/chapter-10/making-a-payment-7.tsx:14]:
```suggestion
export default function MakingAPayment7({ lang }) {
```

**Review by benalleng — APPROVED** [review]

File: content/lessons/chapter-10/updating-the-state-5.tsx [hunk 3]
```diff
@@ -18,33 +17,43 @@ export default function UpdatingTheState5({ lang }) {
...
-        output_0: ['OP_PUSH'],
+        output_0: [
+          'OP_IF',
+          'OP_PUSH',
+          'OP_CHECKSEQUENCEVERIFY',
+          '700',
+          'PUBKEY(REVOCATION_YOU_1)',
+          'OP_ELSE',
+          'OP_CHECKMULTISIG',
+        ],
```
Comment (pinheadmz) [inline content/lessons/chapter-10/updating-the-state-5.tsx:35]: I think this script has a few mistakes in it.

OUTPUTS:
```
  Amount   |   Script
-----------|---------------------------
  100000   |  OP_IF
           |    OP_PUSH 700 OP_CHECKSEQUENCEVERIFY OP_DROP
           |    OP_PUSH PUBKEY(You) OP_CHECKSIG
           |  OP_ELSE
           |    OP_2 OP_PUSH PUBKEY(revocation_you_1) OP_PUSH PUBKEY(Bob) OP_2 OP_CHECKMULTISIG
           |  OP_ENDIF
```

Comment (benalleng) [reply content/lessons/chapter-10/updating-the-state-5.tsx:35]: This is just a `.includes()` mehtod so while not every script is present the error you were running into was actually 13 lines up where `1: ['SIG(YOU) 1]` needed to be `1: ['SIG(YOU)', '1']`

**Pull request merged**